crates/wasmedge-sys/build.rs

```diff
@@ -7,14 +7,18 @@ use build_paths::{Env, LibWasmEdgePaths};
 mod build_standalone;
 use build_standalone::*;
 
+use crate::build_paths::AsPath;
+
 const WASMEDGE_RELEASE_VERSION: &str = "0.13.3";
 const REMOTE_ARCHIVES: phf::Map<&'static str, (&'static str, &'static str)> = phf_map! {
```

Comment:
Could you please add some comments to explain what the key and value stand for, respectively? Thanks.